src/main/java/com/academy/beerinventory/web/service/IBeerService.java

```diff
@@ -1,18 +1,19 @@
 package com.academy.beerinventory.web.service;
 
 import com.academy.beerinventory.domain.Beer;
+import com.academy.beerinventory.web.model.BeerDto;
 
 import java.util.List;
 
 public interface IBeerService {
 
-    public List<Beer> findAll();
+    public List<BeerDto> findAll();
 
-    public Beer findById(Long id);
+    public BeerDto findById(Long id);
 
-    public Beer addBeer(Beer beer);
+    public BeerDto addBeer(Beer beer);
```

Comment:
Lo mismo los parámetros que tomas en addBeer and updateBeer tienen que ser BeerDto(que es lo que te manda el user) y convertirlo en el servicio a Beer para guardarlo/actualizarlo en BBDD